symbolic-base/test/Tests/Arithmetization/Optimization.hs

```diff
@@ -53,7 +53,7 @@ specOptimization' = do
         it "number of constraint decreases" $ do
             acSizeN ac === 1
         it "number of variables decreases" $ do
-            acSizeM ac === 1
+            acSizeM ac === 0
```

Comment:
So actually `optimize` does nothing here because of `LinVars`
Maybe add a single multiplication at the start of `testFunc`/`testConst` to stop the linear optimization from stealing all the work from `optimize`?

Reply:
(I'd add this as another circuit and another bunch of tests instead of replacing the original tests, maybe)